build-scripts/bundle.cjs

```diff
@@ -69,8 +99,16 @@ module.exports.htmlMinifierOptions = {
   decodeEntities: true,
   removeComments: true,
   removeRedundantAttributes: true,
-  minifyCSS: {
-    compatibility: "*,-properties.zeroUnits",
+  minifyCSS: (text, type) => {
+    const input = wrapCSS(text, type);
+    const { code } = lightningcss.transform({
```

Comment:
Should also collect the warnings array here from output, and log them if not empty

Reply:
Hmm. I get warnings about `Unsupported pseudo class or element: b2lopmc8akq` whenever someone uses a variable inside of a css template literal, because one of the plugins replaces all variables with things like `babel-plugin-template-html-minifier:b2lopmc8akq`.

Reply:
That plugin is what's running `html-minifier-terser` on all the tagged templates.  For debugging (if you already haven't), do dev builds with:
1. Comment out the `isProdBuild &&` in front of that plugin in the Babel options
2. Change the `devTool` setting in `webpack.cjs` to get rid of `eval-`

After that, it should be much easier to find and inspect various templates to see what's happening.  Wherever there are substitutions, technically there should be nothing but numbers or other CSS templates because that's all lit allows without `unsafeCSS`.

For `@apply`, I think you can add exceptions in the settings?